client/dist/styles.css

```diff
@@ -2,11 +2,12 @@
   box-sizing: border-box;
 }
 
```

Comment:
/*
Clamping our HTML here will define the minimum font size unit for our app when shrunken and when full-screen.
All children will have > 1 rem. That way, they'll never be smaller than the minimum bounds we're defining here.

MIN should be the minimum font size for all elements when shrunken
VALUE should be the minimum font size for all elements when full-screen
MAX doesn't matter too much right now
*/
html {
  font-size: clamp(8px, 0.6vw, 24px);
}